src/c++/perf_analyzer/client_backend/client_backend.cc

```diff
@@ -60,7 +56,7 @@ Error::Error(const std::string& msg, const uint32_t err)
 
 Error::Error(const std::string& msg) : msg_(msg)
 {
-  error_ = DEFAULT_ERROR;
+  error_ = GENERIC_ERROR;
```

Comment:
Sorry to drag it out once more, but if the default error code is set here, technically most of the changes in the PR can be reverted, since specifying the `pa::GENERIC_ERROR` is redundant. Up to you if you wanna make this change.

Reply:
ill leave them explicit for now but its a good call out.